diablo/models/sis_section.py

```diff
@@ -218,8 +218,8 @@ def get_course_changes(cls, term_id):
             scheduled = course['scheduled']
             if scheduled['hasObsoleteRoom'] \
                     or scheduled['hasObsoleteInstructors'] \
-                    or scheduled['hasObsoleteMeetingDates'] \
-                    or scheduled['hasObsoleteMeetingTimes']:
+                    or scheduled['hasObsoleteDates'] \
+                    or scheduled['hasObsoleteTimes']:
```

Comment:
Having the word 'meeting' in these keys was a bit misleading. The dates describe Kaltura schedule, not course meetings.